main.go

```diff
@@ -26,6 +26,7 @@ type config struct {
 	DepWarnning  string
 	Script       []string `yaml:"script"`
 	Build        []string `yaml:"build"`
+	Runners      []string `yaml:"runners"`
```

Comment:
I'm not too found of the name either. Not too sure what to call it. This is an `after`/ `run` hook.